PyCytominer, CytoTable, CytoSnake

d33bs opened the pull request:
This PR adds Cytomining projects Pycytominer, CytoSnake, and CytoTable to the DBMI Wall of Software. These are added with white backgrounds based on the original PNG's. I used [ImageMagick](https://imagemagick.org/) to convert the png's into jpg's (for ex: `$ convert <project>.png -background white -alpha remove -alpha off <project>.jpg`.)

As part of this process I noticed a small change needed for the PR template (a space for the checkmarks to allow them to properly render as checkboxes).

Thanks in advance for any thoughts / feedback you may have!

CC @gwaybio, @axiomcura

Submission/review checklist:

- [x] I have followed the submission guidelines in the `README.md`.
- [x] I have looked at the preview link to verify that my changes appear correctly.

d33bs committed:
add cytomining project jpg's

Co-Authored-By: Vincent Rubinetti <8326331+vincerubinetti@users.noreply.github.com>

d33bs committed:
adding cytomining project json additions

d33bs committed:
update pr template for space in brackets for list

vincerubinetti commented:
Hmm the outlines of those particular logos don't look so great in the hexagons. Would you mind doing a simple color fill around them.

Edit: Next week I'll fix that little outline of pixels leaking outside the hexagons.

Also could you put the new ones at the top of the list. I figured we'd have people always add to the top so we have it in order of recency (even though we shuffle them randomly on page load, maybe someday we'll want to show the recency somehow).

d33bs committed:
add image padding for hex displays

d33bs committed:
update json list order

d33bs committed:
add more padding to images

d33bs commented:
Thanks @vincerubinetti ! I've adjusted the order of the JSON items and added some padding to the image icons (they seem to appear in the hex properly now). As a reference point I also used ImageMagick for this via `convert <project>.jpg -bordercolor white -border 20x20 <project>.jpg`.

d33bs commented:
Thanks @vincerubinetti ! Going to merge this in. Open to adjustments if needed in the future 🙂
